guides/common/modules/proc_distributing-keys-for-remote-execution.adoc

```diff
@@ -1,7 +1,9 @@
 [id="Distributing_SSH_Keys_for_Remote_Execution_{context}"]
 = Distributing SSH Keys for Remote Execution
 
-To use SSH keys for authenticating remote execution connections, you must distribute the public SSH key from {SmartProxy} to its attached hosts that you want to manage.
+For {SmartProxies} in `ssh` mode, remote execution connections may be authenticated using SSH.
```

Comment:
"may be"?

Reply:
That was because of the sentence below
```
If the hosts are running on Amazon Web Services, enable password authentication.
```

Reply:
Unless amazon is doing something odd in there, the authentication will always be handled by ssh, no matter if you use a public key or password based auth, no?

Reply:
ahh gotcha, I was just a bit confused and thought "password authentication" implied non-SSH somehow. Will update